starboard/tools/testing/test_runner.py

```diff
@@ -184,6 +197,28 @@ def Kill(self):
   def Join(self):
     self.runner_thread.join()
 
+  def _Run2(self) -> None:
+    """Runs the launcher, and assigns a return code.
+    Sets return_code to the following values:
+      - 0x00-0x03F - target return code;
```

Comment:
This seems to be attempting to pack two different things into one variable - native return code, and extra signaling if the result was a crash, launch failed, or could not be determined.

Is there a reason to try to this through a single variable ( return code ) rather than two separate ones ( or a single one that is a tuple ) ? 

E.g. `native_return_code` that yields whatever the OS returned and an enum that just list the statuses that the launcher was able to map ( OK, LAUNCH_FAILED, CRASH, NOT_DETERMINED )

Reply:
TestLauncher.return_code now is a tuple of type (ReturnCodeStatus, int).
TestLauncher.GetReturnCode also returns this type.